src/Core/Content/ProductExport/ScheduledTask/ProductExportGenerateTaskHandler.php

```diff
@@ -73,55 +65,117 @@ public function run(): void
      */
     private function fetchSalesChannelIds(): array
     {
-        $criteria = new Criteria();
-        $criteria
-            ->addFilter(new EqualsFilter('typeId', Defaults::SALES_CHANNEL_TYPE_STOREFRONT))
-            ->addFilter(new EqualsFilter('active', true));
-
-        return $this->salesChannelRepository
-            ->searchIds($criteria, Context::createCLIContext())
-            ->getIds();
+        $salesChannelIds = $this->connection->fetchFirstColumn(
+            <<<'SQL'
+                SELECT `id`
```

Comment:
```suggestion
                SELECT LOWER(HEX(`id`))
```

Then you don't need to use the rest of array transformation below